src/js/apps/admin/sidebar/clinician-sidebar_app.js

```diff
@@ -15,18 +15,18 @@ export default App.extend({
   viewEvents: {
     'save': 'onSave',
     'close': 'stop',
-    'delete': 'onDelete',
+    'delete:clinician': 'onDeleteClinician',
   },
   onSave({ model }) {
     this.clinician.save(model.attributes).done(() => {
       Radio.trigger('event-router', 'clinician', this.clinician.id);
     });
   },
-  // onDelete() {
-  //   this.clinician.trigger('editing', false);
-  //   this.clinician.destroy();
-  //   this.stop();
-  // },
+  onDeleteClinician() {
+    this.clinician.trigger('editing', false);
```

Comment:
I don't think this trigger is needed

Reply:
I think this is still true.  the destroy should remove it from the collectionview that would listen to this trigger, so I'm not sure it is needed.

Reply:
I also noticed in the `onStop` there's a `this.clinician &&` that isn't needed.  If it is needed the `trigger` on the line before would error.